src/app/pages/forms-and-validation/sectioned-form/sectioned-form.component.scss

```diff
@@ -0,0 +1,196 @@
+@import '~@pxblue/colors/palette.scss';
```

Comment:
Everything looks great. 

I'm struggling to understand why we need so much CSS though. This is 200 lines of CSS for a pretty simple form. @emclaug2 is this typical for an Angular application? I feel like I could build this with like three classes in React.

Reply:
I think the amount of css is fine for this example since it's responsive on mobile viewports.   I would have probably used for flexes instead of width 50% where possible but overall it looks good.